packages/studio/src/components/AddPageButton/EntityIdField.tsx

```diff
@@ -0,0 +1,138 @@
+import Select, { MultiValue, StylesConfig } from "react-select";
+import useStudioStore from "../../store/useStudioStore";
+import StreamScopeFieldLabel from "./StreamScopeFieldLabel";
+import { ChangeEvent, useCallback, useMemo, useState } from "react";
+import { pillContainerClass } from "../PillPicker/PillPickerInput";
+import classNames from "classnames";
+
+interface Props {
+  disabled: boolean;
+  updateSelection: (selectedIds: string[]) => void;
+  selectedIds?: string[];
+}
+
+export default function EntityIdField({
```

Comment:
I can add a test for the scrolling! In general I didn't want to add too many tests to this section of the code since I feel like a lot of it could change in the near future

Reply:
I tried a lot of things including overriding the MenuList component to give it an aria-label so I could select it with the testing library, but it looks like react-testing-library does not support scrolling tests due to jsdom not supporting layout

https://github.com/testing-library/react-testing-library/issues/671

Reply:
We can update our acceptance tests later but I don't plan to do that here